@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -124,11 +128,83 @@ export class InterpolatedClipPlanesEvaluator implements ClipPlanesEvaluator {
+        this.m_maxElevation = maxElevation;
+    }
+
+    abstract evaluateClipPlanes(camera: THREE.Camera, projection: Projection): ViewRanges;
+    /**
+     * Set maximum elevation above sea level to be rendered.
+     * @param elevation the elevation (altitude) value in world units (meters).
+     * @note If you set this exactly to the maximum rendered feature height (altitude above
+     * the sea, you may notice some flickering or even polygons disappearing related to rounding
+     * errors or depth buffer precision. In such cases increase [[nearFarMargin]] or add a little
+     * bit offset to your assumed maximum elevation.
+     * @note Reasonable values are in between (-DeadSeeDepression, MtEverestHeight>, both values
+     * are defined in [[EarthConstant]] as [[EarthConstant.MIN_ELEVATION]] and
+     * [[EarthConstant.MAX_ELEVATION]] respectivelly.
+     * @see minElevation for more information about precision and rounding errors.
+     */
+    set maxElevation(elevation: number) {
+        this.m_maxElevation = elevation;
+        // Min elevation should be at least equal or smaller to max elevation.
+        this.m_minElevation = Math.min(elevation, this.m_minElevation);
+    }
+
+    /**
+     * Get maximum elevation to be covered by camera frustum.
+     */
+    get maxElevation(): number {
+        return this.m_maxElevation;
+    }
+
+    /**
+     * Set minimum elevation to be rendered, values beneath the sea level are negative.
+     * @param elevation the minimum elevation (depression) in world units (meters).
+     * @note If you set this parameter to zero you may not see any features rendered if they are
+     * just below the sea level more then half of [[nearFarMargin]] assumed. Similarly if set to
+     * -100m and rendered features lays exactly in such depression, you may notice that problem.
+     * The errors ussually come from projection precision loss and depth buffer nature (significant
+     * precision loss closer to far plane). Thus is such cases either increase the margin (if you
+     * are sure features are just at this elevation, or setup bigger offset for [[minElevation]].
+     * Reasonable values are between <-DeadSeaDepression, MtEverestHeight), where the first denotes
+     * lowest depression on the Earth defined as [[EarthConstants.MIN_ELEVATION]] and the second is
+     * the highest point our planet.
+     * @see https://developer.nvidia.com/content/depth-precision-visualized
+     */
+    set minElevation(elevation: number) {
+        this.m_minElevation = elevation;
+        // Max elevation should be at least equal or bigger then min elevation.
+        this.m_maxElevation = Math.max(elevation, this.m_maxElevation);
+    }
+
+    /**
+     * Get minimum elevation to be covered by camera frustum.
+     */
+    get minElevation(): number {
+        return this.m_minElevation;
+    }
+}
+
 /**
  * Simple camera clip planes evaluator that computes near plane based on camera to ground distance.
```

Comment:
Please update the description. This evaluator is far from simple

Reply:
Updated.